fixed typo, that should say 'in' not 'from' (#36046)
vercel/next.js: docs/advanced-features/react-18/server-components.md

For client components, append `.client.js` to the filename. For example, `./components/avatar.client.js`.

> You can then import other server or client components in any server component. Note: a server component **can not** be imported by a client component. Components without "server/client" extensions will be treated as shared components and can be used and rendered by both sides, depending on where it is imported. For example:

Reviewer: I’m thinking we should flip these sentences around. Something like “A server component can import both server components and client components. However, client components cannot import server components.

> You can then import other server or client components in any server component. Note: a server component **can not** be imported by a client component. Components without "server/client" extensions will be treated as shared components and can be used and rendered by both sides, depending on where it is imported. For example:

Reviewer 2: ```suggestion
Server components can import server components and client components. 

Client components **cannot** import server components. 

Components without a `server` or `client` extension will be treated as shared components and can be imported by server components and client components. For example:
```
